tests/integration/__init__.py

```diff
@@ -0,0 +1,59 @@
+import luigi
+import os
+import pandas as pd
+import sqlalchemy
+import sys
+if sys.version_info[0] < 3:
+    from StringIO import StringIO
+else:
+    from io import StringIO
```

Comment:
Is this the more pythonic way? I thought the standard way of doing this was

```python
try:
    import x
except ImportError:
    import y
````

Reply:
Yes, that is more Pythonic. Stole the line from SO without thinking about that.